tests/unit/fauxjax-testdone-test.js

```diff
@@ -0,0 +1,206 @@
+/*global ok:true */
+import TestDone from "ember-cli-test-helpers/tests/helpers/test-done";
```

Comment:
This will make life much better for ajax related stub mismatch errors! nice work !!!